tensorboard/secured/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# docker build -t activeeon/tensorboard .
+FROM activeeon/tensorflow
+
+RUN mkdir /logs
+
+RUN pip install -U pip
+RUN pip install -U tensorboard_plugin_profile
+
+RUN apt-get update
+RUN apt-get install -y apache2-utils
+RUN apt-get install -y nginx
+
+ADD nginx_docker.conf /etc/nginx/sites-available/nginx.conf
+
+ENV PORT=6009
+ENV LOGIN="admin"
+ENV PASSWORD="admin"
```

Comment:
does that mean that anyone can use these default credentials?

Reply:
No @mboussaa. These are default values that are never used. The main goal of the use of this docker image and not the basic one is to set these env variables (in the run command). If these laters are not set, I chose to put admin admin, but it could be empty value aswell. 
In any case, in our WFs, if a user does not define a LOGIN and PASSWORD, we will use the basic docker image (without NGINX). :)